#! /bin/sh /usr/share/dpatch/dpatch-run
## gtk-clist.patch by Ed Catmur
##
## DP: This patch closes bug #518633  
## DP: It is taken from https://bugs.launchpad.net/inkscape/+bug/238223

@DPATCH@

--- inkscape-0.46/src/widgets/sp-xmlview-attr-list.h	2008/06/07 22:06:52	1.1
+++ inkscape-0.46/src/widgets/sp-xmlview-attr-list.h	2008/06/07 22:09:22
@@ -13,7 +13,7 @@
  */
 
 #include <stdio.h>
-#include <gtk/gtkclist.h>
+#include <gtk/gtk.h>
 #include "../xml/repr.h"
 
 #include <glib.h>

